packages/affiliates/liquidity-mining/yusdeth-weekly-payouts/Week_22_Mining_Rewards.json

```diff
@@ -0,0 +1,265 @@
+{
+  "week": 22,
+  "fromBlock": 11499618,
+  "toBlock": 11545304,
+  "poolAddress": ["0xCce41676a4624F4A1e33a787A59d6bF96e5067bc", "0x5e065D534d1DAaf9E6222AfA1D09e7Dac6cbD0f7"],
+  "blocksPerSnapshot": 256,
+  "rollNum": 3,
+  "nonOverlapUma": 7749,
+  "OverlapUma": 5865,
+  "startRollBlock": 11545304,
+  "umaPerWeek": 13614,
+  "shareHolderPayout": {
```

Comment:
This is the final shareholder payout. UMA liquidity miners are the numbers that matter & correspond to the rewards you will recive. If your address appears multiple times it's due to the roll payouts to enable auditability.